pom.xml

```diff
@@ -1556,78 +1000,6 @@ under the License.
 							</rules>
 						</configuration>
 					</execution>
-					<execution>
-						<id>ban-unsafe-snakeyaml</id>
-						<goals>
```

Comment:
Did you verify that the connector does not use jackson, snakeyaml or uses versions that are not vulnerable?

Reply:
I haven't. Been I'm also wondering if that still makes sense for an external connector: there are more dependencies of course that could be vulnerable, why specifically target these?

Reply:
I think these are less important for connectors because the project is much more focused, with a relatively small dependency tree that also doesn't change a lot.
The dependency tree of Flink is massive, so such safeguards make more sense there.